packages/test/test-service-load/src/runner.ts

```diff
@@ -444,6 +513,9 @@ function scheduleOffline(
 				const injectionTime = random.integer(offlineDelayMinMs, offlineDelayMaxMs);
 				await new Promise<void>((resolve) => setTimeout(resolve, injectionTime));
 
+				if (container.closed) {
```

Comment:
i think i would make changes below here by calling closeAndGetPendingState if the container has offline enabled for some set of clients